### DISCUSSION TITLE
devtools-ui-kit vs nuxthq/ui

### DISCUSSION THREAD

**Discussion by majordomo-admin** [nuxt/devtools#233] [Q&A] [answered]
Are these two UI kits iterations of the same project? Is nuxthq/ui the precursor to the devtools version?  I notice UButton vs NButton etc and other changes.

Just wondering which I should use

**Comment by MuhammadM1998** [comment, marked as answer]
Both are for different usecases

devtools-ui-kit (as stated in the docs)[https://devtools.nuxtjs.org/module/ui-kit] is for module authors to build the custom view with the similiar look and feel as built-in DevTools UI. So if you want to build a module that integerates into Nuxt Devtools, you should use devtools-ui-kit to make your module match the identity of nuxt devtools.

nuxthq/ui is a UI Components Library for Nuxt apps, If you're building a webspite, app, blog, etc.. you would use nuxthq/ui

**Comment by majordomo-admin** [comment]
Thanks for the answer!
